docker/config.hornet.json

```
@@ -243,4 +243,4 @@
       "workerCount": 0
     }
   }
-}
+}
```

Please add missing newline.

```suggestion
}

```

---

Why? My IDE seems to disagree.

---

your IDE is probably not configured to add a blank line? Or Gh is lying to us.